Mathlib/Tactic/Linarith/SimplexAlgo/Datatypes.lean

```diff
@@ -0,0 +1,48 @@
+/-
+Copyright (c) 2024 Vasily Nesterov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Vasily Nesterov
+-/
+import Std.Data.Rat.Basic
+
+/-!
+# Datatypes for Simplex Algorithm implementation
+-/
+
+namespace Linarith.SimplexAlgo
+
+/--
+Structure for matrices over ℚ.
+
+So far it is just a 2d-array carrying dimensions (that supposed to coincide with the actual
+dimensions of `data`), but the plan is to add some `Prop`-data and make the structure strict and
+safe.
+
+Note: we avoid using `Mathlib.Data.Matrix` because it is far more efficient to store matrix as its
```

Comment:
```suggestion
Note: we avoid using the `Matrix` in the root namespace because it is far more efficient to store matrix as its
```
which will create a link in the docs to the definition.